fair/src/autogluon/fair/learners/fair.py

```diff
@@ -0,0 +1,635 @@
+                                                      groups,
+                                                      val_thresholds.argmax(1), self.frontier[1])
+            front2 = efficient_compute.compute_metric(objective2, labels, proba,
+                                                      groups,
+                                                      val_thresholds.argmax(1), self.frontier[1])
+
+            zero = [objective1(labels, proba.argmax(1), groups),
+                    objective2(labels, proba.argmax(1), groups)]
+
+        plt.scatter(front2, front1, label='Frontier')
+        plt.scatter(zero[1], zero[0], label='Original predictor')
+        if np.any(self.offset):
+            if self.use_fast is False:
+                front1 = fair_frontier.compute_metric(objective1, labels, proba,
+                                                      val_thresholds, self.offset[:, :, np.newaxis])
+                front2 = fair_frontier.compute_metric(objective2, labels, proba,
+                                                      val_thresholds, self.offset[:, :, np.newaxis])
+            else:
+                front1 = efficient_compute.compute_metric(objective1, labels, proba, groups,
+                                                          val_thresholds.argmax(1),
+                                                          self.offset[:, np.newaxis])
+                front2 = efficient_compute.compute_metric(objective2, labels, proba, groups,
+                                                          val_thresholds.argmax(1),
+                                                          self.offset[:, np.newaxis])
+            plt.scatter(front2, front1, label='Updated predictor')
+        plt.legend(loc='best')
+
+    def evaluate(self, data=None, metrics=None) -> pd.DataFrame:
+        """Compute standard metrics of the original predictor and the updated predictor
+         found by fit and return them in a dataframe.
+          If fit has not been called only return the metrics of the original predictor.
+        parameters
+        ----------
+        data: (optional) a pandas dataframe to evaluate over. If not provided evaluate over
+            the dataset provided at initialisation.
+        metrics: (optional) a dictionary where the keys are metric names and the elements are either
+                    scoreables or group metrics. If not provided report the standard metrics
+                    reported by autogluon on binary predictors
+        returns
+        -------
+        a pandas dataset containing rows indexed by metric name, and columns by
+        ['original', 'updated']
+         """
+        if metrics is None:
+            metrics = group_metrics.ag_metrics
+
+        groups = None
+        if data is not None:
+            groups = np.ones(data.shape[0])
+
+        return self.evaluate_fairness(data, groups, metrics)
+
+    def evaluate_fairness(self, data=None, groups=None, metrics=None) -> pd.DataFrame:
```

Comment:
Similar to my rendering comment.  Can we make it so that the user can see the definition of each metric by "clicking on it" using the `IPython.display` library?